pyanaconda/modules/storage/installation.py

```diff
@@ -337,12 +337,15 @@ def _write_lvm_devices_file(self, storage, sysroot):
 
         Adds all present PVs according to https://bugzilla.redhat.com/show_bug.cgi?id=2011329#c9
 
+        /etc/lvm/devices/system.devices
+
         :param Blivet storage: instance of Blivet or a subclass
         :param str sysroot: path to the target OS installation
         """
-        for device in storage.devices:
-            if device.format and device.format.type == "lvmpv" and HAVE_LVMDEVICES:
-                device.format.lvmdevices_add()
+        for devices in (storage.devices, storage.devicetree._hidden):
```

Comment:
You can use `itertools.chain` to iterate over both lists of devices.

Reply:
Sounds good! I believed you would want it the other way, less imports :-)

Reply:
Imports from standard libraries are never an issue :-)